core/importer/datasets_importer.py

```diff
@@ -149,7 +149,7 @@ def process_data_locations(self, dataset, dataset_dict):
                 category = self.process_location_category(storage_location_dict)
 
 
-                location_delimeted = '\n'.join(storage_location_dict['locations'])
+                location_delimeted = '\n'.join(storage_location_dict['location'])
```

Comment:
If it's going to be a string (and not dict anymore), `'\n'.join()` might be removed

Reply:
but it's no biggie, as string.join(X) works for both strings and iterables

Reply:
yes! I wanted to keep that - I am not sure we wont go back to list/dict at some point ;) 
Thanks!